### Coin Flip  

  Excerise 1.
  
  Note that there are the same number of ways to get 4 tails as there to get 4 heads. Explain why this is?
  


In [6]:

(10 * 9 * 8 * 7) // (4 * 3 * 2 * 1)


210

There is 210 possibilities of getting 4 heads as their is to 4 tails.
This is because their are 10 coins that are flipped. This means that the first head can land in any on the flips from 1 - 10. This leaves nine chances for the head to land in the second flip. Eight for the 3rd and 7 for the fourth. Because we are looking for four heads we stop their. This is double counting so we then divide then by the (4 * 3 * 2 * 1) to get rid of the duplicate counts


Excerise 2.

It is somewhat interesting that (5 * 4 * 3 * 2 * 1) perfectly divides (10 * 9 * 8 * 7 * 6) - there's no remainder.
If we only wanted exactly four heads as opposed to five, the equivalent calculation would be (10 * 9 * 8 * 7) / (4 * 3 * 2 * 1).
Does that evenly divide too? What is the formula in general?
Does it always come out as a positive whole number?



In [8]:
(10 * 9 * 8 * 7) // (4 * 3 * 2 * 1)

210

Yes it divides into it evenly.